packages/lwc-language-server/src/__tests__/component-indexer.test.ts

```diff
@@ -125,6 +126,54 @@ describe('ComponentIndexer', () => {
                 expect(componentIndexer.tags.size).toBe(8);
             });
         });
+
+        describe('typescript path mapping', () => {
+            const expectedComponents: string[] = [
+                'force-app/main/default/lwc/*/hello_world',
+                'force-app/main/default/lwc/*/import_relative',
+                'force-app/main/default/lwc/*/index',
+                'force-app/main/default/lwc/*/lightning_datatable_example',
+                'force-app/main/default/lwc/*/lightning_tree_example',
+                'force-app/main/default/lwc/*/todo_item',
+                'force-app/main/default/lwc/*/todo',
+                'force-app/main/default/lwc/*/typescript',
+                'force-app/main/default/lwc/*/utils',
+                'utils/meta/lwc/*/todo_util',
+                'utils/meta/lwc/*/todo_utils',
+            ].map(item => path.join(componentIndexer.workspaceRoot, item));
+
+            describe('#tsConfigPathMappingFiles', () => {
+                it('returns a list of files where the .js or.ts filename is the same as its parent directory name', () => {
+                    const tsConfigPathMapping = componentIndexer.tsConfigPathMappingFiles.sort();
+                    expect(tsConfigPathMapping).toEqual(expectedComponents.sort());
+                });
+            });
+
+            describe('updateSfdxTsConfigPath', () => {
+                it('updates updates tsconfig.sfdx.json path mapping', async () => {
```

Comment:
This looks like a typo.

```suggestion
                it('updates tsconfig.sfdx.json path mapping', async () => {
```